packages/babel-cli/test/index.js

```diff
@@ -174,7 +174,9 @@ fs.readdirSync(fixtureLoc).forEach(function (binName) {
         opts.inFiles[".babelrc"] = helper.readFile(babelrcLoc);
       }
 
-      it(testName, buildTest(binName, testName, opts));
+      if (testName !== "flag-dashed-inspect" || !process.version.match(/^v[045]/)) {
```

Comment:
I want to exclude the test for  `--inspect` in Node v0.12, v4 and v5.
I doesn't like this workaround, but I couldn't find better way.